### PR TITLE
[S2GRAPH-9] Provide rest server using netty

### PR DESCRIPTION
**Pull request by daewon**
This issue dependence on S2GRAPH-7 (Abstract common code)

I made Http server using Netty and S2Graph

S2Graph provide CRUD operation, 
but now, read operations were completed

1. Make http rest server using netty
2. Attach s2core to netty server and refactoring some s2rest_play's controllers using `RestHandler`

### CODE REVIEW

File: s2core/src/main/scala/com/kakao/s2graph/core/PostProcess.scala [hunk 1]
```diff
@@ -323,10 +323,17 @@ object PostProcess extends JSONParser {
       to <- innerValToJsValue(edge.tgtVertex.id.innerId, tgtColumn.columnType)
     } yield {
       val targetColumns = if (q.selectColumnsSet.isEmpty) reservedColumns else (reservedColumns & q.selectColumnsSet) + "props"
+<<<<<<< HEAD:s2core/src/main/scala/com/kakao/s2graph/core/PostProcess.scala
```
Comment (hsleep) [inline s2core/src/main/scala/com/kakao/s2graph/core/PostProcess.scala]: It has been merged incorrectly.

**Comment by daewon** [comment]
@hsleep sorry, fixed

**Comment by hsleep** [comment]
:+1: Looks good to me.

**Comment by SteamShon** [comment]
@daewon  so it seems that this PR is resolving S2GRAPH-9 and S2GRAPH-27 at the same time. I think it is better for us to make issue and related PR small to make reviewer`s life easier. But I guess this time, it seems reasonable to resolve two issue at the same time. please close S2GRAPH-27 also when you close this issue. PR itself looks good to me. +1

**Comment by djfwan** [comment]
(+1) Looks good.

**Comment by emeth-kim** [comment]
+1 LGTM

**Comment by SteamShon** [comment]
+1, passed all TCs. nice works!

**Pull request closed without merging**